docs/resource-cleanup.md

```diff
@@ -0,0 +1,85 @@
+# Stale Resource Cleanup
+
+If the PVC is created with storage class which is having the `reclaimPolicy`
+as `Retain` will not delete the PV object, backend omap metadata and backend image.
+Manual deletion of PV will result in stale omap keys, values, cephfs subvolume and rbd image.
+It is required to cleanup metadata and image separately.
+
+## Steps
+
+
+### 1. Get pv_name from pvc_name
```

Comment:
```suggestion
Get PV name from PVC
```

Reply:
and also just a question, is it possible to get the PV name when PVC is deleted?

Reply:
It doesn't seem to be possible without following a naming convention.